ios/Gemfile

```diff
@@ -1,3 +1,3 @@
 source "https://rubygems.org"
 
-gem "cocoapods", "1.14.3"
+gem "cocoapods", "1.15.2"
```

Comment:
this should be set also in the scripts and then checked in solidarity

Reply:
It turns out there is no need for this, I reverted it